api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -9534,5 +9534,13 @@
             AND NOT EXISTS (SELECT role, privilege FROM role_privilege rp2 WHERE rp2.privilege='Add Visits'                     AND rp2.role=rp.role);
         </sql>
     </changeSet>
+	<changeSet id="TRUNK-4936-20160930-1000" author="teleivo">
+		<comment>Add unique_id column to concept_reference_source</comment>
```

Comment:
added it at https://github.com/openmrs/openmrs-core/pull/1847

is it necessary to change the changeSet id  ? I left it as is in the current PR.